alembic/versions/d0047bf55eac_eq.py

```diff
@@ -0,0 +1,431 @@
+"""EQ migration
+
+Revision ID: d0047bf55eac
+Revises: 2c856b2845f2
+Create Date: 2022-09-02 02:50:05.173028
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd0047bf55eac'
+down_revision = '2c856b2845f2'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'earthquakeevents',
+        sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('modified', sa.DateTime(), nullable=False),
+        sa.Column('sent_by_id', sa.Integer(), nullable=False),
+        sa.Column('event_id', sa.String(), nullable=False, comment='Earthquake ID'),
+        sa.Column('event_uri', sa.String(), nullable=True, comment='Earthquake URI'),
```

Comment:
Should probable change this to a sa util `URLType` (https://sqlalchemy-utils.readthedocs.io/en/latest/data_types.html#module-sqlalchemy_utils.types.url) instead of sa.String type.  I checked that URIs are properly parsed:

```python
import furl
furl.furl("ivo://nvo.caltech/voeventnet/catot#1004071150784109051").scheme
'ivo'
```

Reply:
Added!